examples/06_assets/main.rs

```diff
@@ -1,66 +1,32 @@
 extern crate amethyst;
 extern crate cgmath;
-extern crate obj;
 
-use amethyst::engine::{Application, State, Trans};
-use amethyst::context::{ContextConfig, Context};
 use amethyst::config::Element;
-use amethyst::ecs::{World, Join};
-use amethyst::context::asset_manager::{Assets, AssetLoader, AssetLoaderRaw, DirectoryStore, Mesh, Texture};
-use amethyst::renderer::VertexPosNormal;
-use cgmath::{InnerSpace, Vector3};
-use std::io::BufReader;
-
-struct Obj(obj::Obj);
```

Comment:
I feel like there should still be an example showing how to implement a custom asset loader. Or maybe it's too trivial